libs/core/src/decorators/debounce.decorator.spec.ts

```diff
@@ -0,0 +1,184 @@
+import { Component } from '@angular/core';
+import {
+  fakeAsync,
+  tick,
+} from '@angular/core/testing';
+
+import { debounce } from './debounce.decorator';
+
+describe('debounce decorator', () => {
```

Comment:
```suggestion
describe('@daffodil/core | debounce', () => {
```

This is the standard way that we do top level describes.